server/data/weidu-tp2-ielib.yml

```diff
@@ -3803,6 +3803,9 @@ ielib-resref:
     - name: WIZARD_REMOVE_MAGIC
       detail: resref WIZARD_REMOVE_MAGIC = "SPWI302"
       doc: IElib define
+    - name: WIZARD_DISPEL_MAGIC
+      detail: resref WIZARD_DISPEL_MAGIC = "SPWI302"
+      doc: IElib define
```

Comment:
You can submit ielib changes directly, they have to from from ielib.

Reply:
> You can't submit ielib changes directly, they have to come from ielib.

Removed.